packages/react-router-native/package.json

```diff
@@ -26,6 +27,8 @@
     "babel-preset-react-native": "1.9.1",
     "jest": "18.1.0",
     "react": "15.4.1",
+    "react-addons-test-utils": "^15.4.2",
+    "react-dom": "^15.4.2",
```

Comment:
I don't think we need react-dom. If there are peer dep errors, you can ignore them.